### ISSUE TITLE
[Ch3] Containers under-the-hood (runc, containerd, containerd-shim)

### ISSUE THREAD

**Issue by min9pm8** [mhausenblas/the-container-security-book#4]
Within container basics, we need to include sections on open source components that run within a Docker process, and how this relates to the history around container runtimes (high-level and low-level), efficiencies and security issues (e.g. runc CVE).

**Comment by mhausenblas** [comment]
Yup, that should be in the [runtime](https://github.com/mhausenblas/the-container-security-book/blob/master/manuscript/ch3.md#container-runtimes) section, or?
